backend/compact-connect/docs/onboarding/JURISDICTION_COMPACT_ONBOARDING.md

```diff
@@ -0,0 +1,170 @@
+# Jurisdiction Onboarding for Compact Connect
+
+Before a jurisdiction (i.e. state) can be onboarded to the Compact Connect system, there is certain information that 
+must be provided for that jurisdiction. This document outlines the information that is required and how that information
+is to be defined in the system. The following steps must be taken to onboard a jurisdiction to the Compact Connect system:
+
+1. Decide Jurisdiction Fee for a Compact Privilege
+2. Determine Contact Details for System Notifications
+3. Ensure Jurisdiction License Names and Adverse Actions are compatible with System
+4. Determine Jurisprudence Requirements
+5. Determine Active Development Environments for Jurisdiction
+6. Add Jurisdiction Configuration File to the System
+7. Add Global Compact Configuration File to the System
+
+## Deciding Jurisdiction Fee for a Compact Privilege
+The first step in onboarding a jurisdiction to the Compact Connect system is to decide on the fee that the jurisdiction
+will charge for a compact privilege. The value must be defined as part of the configuration file under the following field.
+```
+jurisdictionFee: number
+```
+
+### Decide Military Discount
+The jurisdiction must also decide if they will offer a military discount for military affiliated licensees. 
+If they choose to do so, that discount amount must be defined.
+```
+militaryDiscount:  
+    discountType: "FLAT_RATE" 
+    active: true 
+    discountAmount: <number>
+```
+
+## Determine Contact Details for System Notifications
+The jurisdiction must provide contact details for the following notifications:
+
+1. Operations Team Notifications
+2. Adverse Actions Notifications
+3. Summary Report Notifications
+
+The following subsections detail the information that must be provided for each notification.
+
+### Operations Team Notifications
+The jurisdiction must provide an email address for the operations team that will be responsible for handling support 
```

Comment:
Unfortunately, I don't have that information currently 😞 . I agree it would be nice to have that information, but until we have a live system with actual use cases for operation support, it really could be either, but any attempt we try to pin down today is going to eventually be updated. If this is to be regarded as a living document, which we actively update and maintain as part of adding new features, then those details will come as we have a better understanding. That is the meaning of agile, right? This is a first iteration of a living document. If you want to flesh this out with a waterfall approach, we can go get every business detail ironed out so that it can be captured in the document today, and we'll still have to update it when management changes requirements on us next month 😄 

So again, I agree, more fleshed out detail will be great. When we reach the point of implementation that gives us a better picture of how the system will send these types of notifications, it can be added to the document.

Reply:
I by no means am implying we should use a waterfall approach :')

That makes sense! I mostly wanted to clarify if the detail was intentionally sparse.

I guess one thought, and this is truly meant to be a question for discussion (I'm not an expert on this): Is including these configuration fields antithetical to the concept of "vertical slicing"? It seems like we're adding configuration for something that does...nothing. Like adding a backend component for a non-existent frontend.

Consider this a rhetorical question for the sake of this PR (i.e., one we don't have to discuss back-and-forth here or resolve to merge), but I thought it might be interesting to discuss.

Reply:
That is a legitimate question, and I'd be curious to get @jusdino's thoughts on it. The initial requirements given to us as part of designing this was to have _something_ in place for email notifications for the three use cases outlined in the document, so I threw them in essentially as a first attempt given we know there _will_ be notifications for these elements, and that's about it 😆 I know we have stories coming up on the backlog for adding email notifications, so the fields will likely be utilized sooner rather than later.

But yeah honestly, in the interest of vertical slicing we could have added them in later, something to consider during our next retrospective.